### PR TITLE
SPCTR-2585-Fix counter initialisation issue in infinite slider loop

### PR DESCRIPTION
**Pull request by chandan-30**
### Description
This PR fixes an issue where the counter does not initialise in the Slider block when it is set to infinite loop. To fix this issue, we update frontend JS code for counter block. This ensures that the counter is properly initialised even when the Slider is set to infinite loop.

### Screenshots
https://d.pr/v/KipZ55

### Checklist:
- [x] My code is tested
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: assets/js/spectra-counter.js [hunk 1]
```diff
@@ -3,6 +3,17 @@ UAGBCounter = {
...
+		const elements = document.querySelectorAll( `.wp-block-uagb-counter${mainSelector}` );
+        if ( elements && elements.length > 1 ) {
+            for( const element of elements ) {
+                this.elements.counterWrapper = element;
+                this.handleCounterWrapper( data );	
+            }
+        } else {
+            this.handleCounterWrapper( data );
+        }   
+	},
```
Comment (supriyajjadhav) [inline assets/js/spectra-counter.js:15]: can we avoid calling the `handleCounterWrapper` function twice? 
```
if ( elements && elements.length > 1 ) {
  for (const element of elements ) {
    this.elements.counterWrapper = element;
  }
}
this.handleCounterWrapper( data );
```

Comment (chandan-30) [reply assets/js/spectra-counter.js:15]: when we add a counter block on first slide in slider block, a duplicate markup of the counter block is being created in DOM, so to make the duplicate markup get observed by Intersection Observer API we are calling handleCounterWrapper function in the loop inside if block.

If there's no duplicate markup of any counter block we are calling handleCounterWrapper function inside else block.

**Review by aapatre — APPROVED** [review]
LGTM!
Works fine ✅ 🚀

**Comment by aapatre** [comment]
Added the changelog, removed eslint comment and declared `UAGBCounter` in globals.

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**